doc_source/introduction-how-it-works.md

```diff
@@ -21,6 +21,7 @@ Global Accelerator supports both TCP and UDP protocols\.
 Be aware of the following when you use Global Accelerator:
 + AWS Direct Connect does not advertise IP address prefixes for AWS Global Accelerator over a public virtual interface\. We recommend that you do not advertise IP addresses that you use to communicate with Global Accelerator over your AWS Direct Connect public virtual interface\. If you advertise IP addresses that you use to communicate with Global Accelerator over your AWS Direct Connect public virtual interface, it will result in an asymmetric traffic flow: your traffic toward Global Accelerator goes to Global Accelerator over the internet, but return traffic coming to your on\-premises network comes over your AWS Direct Connect public virtual interface\.
 + Global Accelerator does not support processing IP packet fragments or re\-assembly\. An intermediate router or gateway operating at layer 3 might fragment a packet into multiple smaller packets between the client and the Global Accelerator endpoint\. If that happens, the fragments are not processed or re\-assembled by Global Accelerator and are not delivered to the endpoint\.
++ Global Accelerator does not support resources/endpoints which reside in different AWS accounts.
```

Comment:
Sounds good to add a line here about this, to make sure people are aware that they can't use resources in multiple AWS accounts. How about this? "Global Accelerator does not support adding as endpoints resources that belong to other AWS accounts."

Reply:
Make sense. Thank you

Reply:
Do you want to update your PR, and then I'd be happy to merge it in?